pxf/pxf-json/src/test/resources/tweets-small.json

```diff
@@ -0,0 +1,4 @@
+{"created_at":"Fri Jun 07 22:45:02 +0000 2013","id":343136547115253761,"text":"REPAIR THE TRUST: REMOVE OBAMA\/BIDEN FROM OFFICE. #IRS #DOJ #NSA #tcot","source":"web","user":{"id":"26643566","name":"SpreadButter","screen_name":"SpreadButter","location":"Austin, Texas"},"entities":{"hashtags":["tweetCongress","IRS"]}}
```

Comment:
better use non-political content for test example.
I think you've fixed it in PDF guide but missed here :)
https://issues.apache.org/jira/secure/attachment/12785045/PXFJSONPluginforHAWQ2.0andPXF3.0.0.pdf